src/bootstrap/format.rs

```diff
@@ -172,4 +277,6 @@ pub fn format(build: &Builder<'_>, check: bool, paths: &[PathBuf]) {
     drop(tx);
 
     thread.join().unwrap();
+
+    update_timestamp(build);
```

Comment:
This should be skipped on `check` since checking doesn't actually update the files.